SCClassLibrary/Common/Control/ServerStatus.sc

```diff
@@ -246,14 +206,17 @@ ServerStatusWatcher {
 		}
 	}
 
+	serverRunning { ^hasBooted and: notified }
+
 	serverRunning_ { | running |
 
-		if(running != serverRunning) {
-			serverRunning = running;
+		hasBooted = running;
+		if(running != server.serverRunning) {
+			hasBooted = running;
 			this.unresponsive = false;
 
-			if (server.serverRunning) {
-				ServerBoot.run(server);
+			if (running) {
+				// serverBoot happens in prRunBootTask now
```

Comment:
if there is nothing in this branch, then why not just write:

```
if(running.not) {

}
```

Reply:
ok, moved the explanatory comment and, and removed the empty true branch.

Reply:
done